cmd/crank/beta/xpkg/init.go

```diff
@@ -235,6 +235,7 @@ func printFile(w io.Writer, path string) error {
 	if _, err := fmt.Fprintf(w, "\n%s\n", content); err != nil {
 		return errors.Wrap(err, "failed to write to stdout")
 	}
+	defer f.Close() //nolint:errcheck // It's safe to ingore the error here.
```

Comment:
This should probably be right after the `os.Open()` so that it will be executed on all of the failure legs.

Reply:
Thanks for the feedback, fix it!

Reply:
Looks like a typo in `ignore`. Also, would be useful to say why it's safe to ignore. I assume because the file was only open for reading, not writing.

Reply:
Yes, I think the same. The whole function is doing read only and not write so I think no need to handle any special case. 

I update the comment, and description in the PR.